File: output/CodeGen/avatar/Java/Python/codeforces_612_B.py
def main():
    s = input()
    n = int(s.next())
    dp = [0] * 200001
    for i in range(n):
        dp[int(s.next())] = i
    ans = 0
    for i in range(2, n+1):
        ans += abs(dp[i] - dp[i-1])
    print(ans)

File: output/ChatGPT/avatar/Java/C++/codeforces_612_B.cpp
#include <iostream>
#include <cmath>
using namespace std;

int main() {
    int n;
    cin >> n;
    
    int dp[200001] = {0};
    for (int i = 0; i < n; i++) {
        int num;
        cin >> num;
        dp[num] = i;
    }
    
    long long ans = 0;
    for (int i = 2; i <= n; i++) {
        ans += abs(dp[i] - dp[i-1]);
    }
    
    cout << ans << endl;
    
    return 0;
}